test/import-map.test.js

```diff
@@ -0,0 +1,82 @@
+const request = require("supertest");
+const { app, setConfig } = require("../src/web-server");
+
+beforeAll(() => {
+  setConfig({
+    manifestFormat: "importmap",
+    locations: {
+      prod: "memory://prod",
+    },
+  });
+});
+
+describe(`/import-map.json`, () => {
+  it(`does not return anything when it's not setup yet.`, async () => {
+    const response = await request(app)
+      .get("/import-map.json")
+      .expect(200)
+      .expect("Content-Type", /json/);
+
+    // we did not setup yet
+    expect(response.body).toMatchObject({ imports: {}, scopes: {} });
+  });
+
+  it(`does give back the same items after first patch`, async () => {
+    const response = await request(app)
+      .patch("/import-map.json")
+      .query({
+        skip_url_check: true,
+      })
+      .set("accept", "json")
+      .send({
+        imports: {
+          a: "/a-1.mjs",
+          b: "/b-1.mjs",
+          c: "/c-1.mjs",
+        },
+      })
+      .expect(200)
+      .expect("Content-Type", /json/);
+
+    // we did not setup yet
```

Comment:
This comment should be removed, as it's a copy pasted message that doesn't make sense here.

```suggestion
```